src/NUnitFramework/framework/Constraints/ConstraintBuilder.cs

```diff
@@ -96,7 +96,7 @@ public ConstraintOperator Pop()
         /// <summary>
         /// ConstraintStack is a type-safe stack for holding Constraints
         /// </summary>
-        public class ConstraintStack
+        public sealed class ConstraintStack
```

Comment:
Is there a reason to not make this internal in-line with the change to OperatorStack?

Reply:
ConstraintOperator has `public abstract void Reduce(ConstraintBuilder.ConstraintStack stack)`.
My bad, `OperatorStack` could have been `private`. I'll change it.